test/include_inliner.cpp

```diff
@@ -93,19 +56,19 @@ class InlinerTest
         const auto header_src      = test_srcs.path / header_filename;
 
         // clang-format-off
-        std::ofstream(valid_src.c_str()) << "#include <" << header_filename << ">" << std::endl
-                                         << "#include \"" << header_filename << "\"" << std::endl
-                                         << "//inliner-include-optional" << std::endl
+        std::ofstream(valid_src.c_str()) << "#include <" << header_filename << ">\n"
+                                         << "#include \"" << header_filename << "\"\n"
+                                         << "//inliner-include-optional\n"
```

Comment:
@apwojcik Can you please remind why the change from `std:endl` to `\n` is necessary? The latter seems non-portable. Thanks.

Reply:
The char `'\n'` is portable between OSes, and compilers convert it correctly (C++ and C specs guarantee that). The use of `'\n'` is recommended instead of `std::endl` for performance reasons. Clang-Tidy can detect that and report a warning. 
The complete explanation is here: https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#slio50-avoid-endl